pkg/api/node.go

```diff
@@ -97,6 +100,8 @@ func (m *nodeMetrics) List(ctx context.Context, options *metainternalversion.Lis
 	for i, node := range nodes {
 		names[i] = node.Name
 	}
+	// maintain the same ordering invariant as the Kube API would over nodes
+	sort.Strings(names)
```

Comment:
Could we move sorting results to ConvertToTable?

I think it make sense to sort when displaying results, here server side table. Sorting every API requests is a overkill, and could cause scalability problems on HPA.